bap-vibes/tests/integration/test_minizinc.ml

```diff
@@ -56,9 +56,9 @@ let (:=) lhs opcode = fun args -> simple_op' opcode lhs args
 let mov = Ir.Opcode.create ~arch:"arm" "mov"
 
 (* Define some temporaries to use in a test. *)
-let t1 = Var.create "t1" (Imm 32)
-let t2 = Var.create "t2" (Imm 32)
-let t3 = Var.create "t3" (Imm 32)
+let t1 = Var.create "_000004ff_t1" (Imm 32)
```

Comment:
I'm wondering how fragile these TID-prefixed names are. Can we trust that they will be consistent across different runs?

Reply:
I don't think they are fragile. I just needed orig_name to return Some instead of none